# Undo the --translate-script-url gating added by core/ungoogled-chromium/toggle-translation-via-switch.patch,
# so translation and the "Translate to" menu work by default; that patch's ranker change stays.
--- a/chrome/browser/renderer_context_menu/render_view_context_menu.cc
+++ b/chrome/browser/renderer_context_menu/render_view_context_menu.cc
@@ -215,7 +215,6 @@
 #include "components/strings/grit/components_strings.h"
 #include "components/supervised_user/core/browser/supervised_user_preferences.h"
 #include "components/supervised_user/core/browser/supervised_user_url_filtering_service.h"
-#include "components/translate/core/common/translate_switches.h"
 #include "components/translate/core/browser/translate_download_manager.h"
 #include "components/translate/core/browser/translate_manager.h"
 #include "components/translate/core/browser/translate_prefs.h"
@@ -2696,7 +2695,6 @@
   }
 
   if (CanTranslate(/*menu_logging=*/true)) {
-   if (base::CommandLine::ForCurrentProcess()->HasSwitch(translate::switches::kTranslateScriptURL))
     AppendTranslateItem();
   }
 }
--- a/components/translate/core/browser/translate_language_list.cc
+++ b/components/translate/core/browser/translate_language_list.cc
@@ -12,7 +12,6 @@
 #include <string_view>
 
 #include "base/check.h"
-#include "base/command_line.h"
 #include "base/containers/fixed_flat_set.h"
 #include "base/containers/flat_set.h"
 #include "base/functional/bind.h"
@@ -32,7 +31,6 @@
 #include "components/translate/core/browser/translate_event_details.h"
 #include "components/translate/core/browser/translate_url_fetcher.h"
 #include "components/translate/core/browser/translate_url_util.h"
-#include "components/translate/core/common/translate_switches.h"
 #include "components/translate/core/common/translate_language_matcher.h"
 #include "components/translate/core/common/translate_util.h"
 #include "net/base/url_util.h"
@@ -276,9 +274,6 @@
 }
 
 void TranslateLanguageList::RequestLanguageList() {
-  const base::CommandLine& command_line = *base::CommandLine::ForCurrentProcess();
-  if (!command_line.HasSwitch(translate::switches::kTranslateScriptURL))
-    return;
   // If resource requests are not allowed, we'll get a callback when they are.
   if (!resource_requests_allowed_) {
     request_pending_ = true;
--- a/components/translate/core/browser/translate_manager.cc
+++ b/components/translate/core/browser/translate_manager.cc
@@ -915,9 +915,8 @@
         TriggerDecision::kDisabledOffline);
   }
 
-  const base::CommandLine& command_line = *base::CommandLine::ForCurrentProcess();
-  if (!command_line.HasSwitch(translate::switches::kTranslateScriptURL) ||
-      (!ignore_missing_key_for_testing_ && !::google_apis::HasAPIKeyConfigured())) {
+  if (!ignore_missing_key_for_testing_ &&
+      !::google_apis::HasAPIKeyConfigured()) {
     // Without an API key, translate won't work, so don't offer to translate in
     // the first place. Leave kOfferTranslateEnabled on, though, because that
     // settings syncs and we don't want to turn off translate everywhere else.
--- a/components/translate/core/browser/translate_script.cc
+++ b/components/translate/core/browser/translate_script.cc
@@ -216,13 +216,8 @@
                         server_params.c_str());
 
     GURL security_origin = translate::GetTranslateSecurityOrigin();
-    const base::CommandLine& command_line = *base::CommandLine::ForCurrentProcess();
-    if (command_line.HasSwitch(translate::switches::kTranslateScriptURL)) {
-      base::StringAppendF(&data_, "var securityOrigin = '%s';\n",
-                          security_origin.spec().c_str());
-    } else {
-      base::StringAppendF(&data_, "var securityOrigin = '';\n");
-    }
+    base::StringAppendF(&data_, "var securityOrigin = '%s';\n",
+                        security_origin.spec().c_str());
 
     // Load embedded translate.js.
     data_.append(ui::ResourceBundle::GetSharedInstance().LoadDataResourceString(
